content/docs/render-props.md

```diff
@@ -4,25 +4,25 @@ title: Render Props
 permalink: docs/render-props.html
 ---
 
-The term ["render prop"](https://cdb.reacttraining.com/use-a-render-prop-50de598f11ce) refers to a technique for sharing code between React components using a prop whose value is a function.
+["Render prop"](https://cdb.reacttraining.com/use-a-render-prop-50de598f11ce) tabiri, React bileşenleri arasında kod paylaşımının; değerleri birer fonksiyon olan prop’lar kullanılarak yapılmasına denir.
 
-A component with a render prop takes a function that returns a React element and calls it instead of implementing its own render logic.
+Render prop’lu bir bileşen, prop olarak bir React elemanı döndüren bir fonksyion alır ve kendi render mantığını yürütmek yerine bu fonksiyonu çağırır.
 
 ```jsx
 <DataProvider render={data => (
   <h1>Hello {data.target}</h1>
 )}/>
 ```
 
-Libraries that use render props include [React Router](https://reacttraining.com/react-router/web/api/Route/render-func) and [Downshift](https://github.com/paypal/downshift).
+Render prop’ları kullanan kütüphaneler arasında [React Router](https://reacttraining.com/react-router/web/api/Route/render-func) ve [Downshift](https://github.com/paypal/downshift) de var.
 
-In this document, we’ll discuss why render props are useful, and how to write your own.
+Bu dokümanda neden render prop’ların kullanışlı olduğunu tartışıp, bunları nasıl yazabileceğiniz hakkında konuşacağız.
 
-## Use Render Props for Cross-Cutting Concerns {#use-render-props-for-cross-cutting-concerns}
+## Render Propları Cross-Cutting İhtiyaçları için Kullanın {#use-render-props-for-cross-cutting-concerns}
```

Comment:
We can translate `cross-cutting concerns` as `uygulama genelini etkileyen özellikler`. What do you think ?

Reply:
I though about `cross-cutting concerns` as a general issue. So I didn't try to translate it so it wouldn't lose its meaning like when you did a google search about it this page would still come accross.